301-multi-tier-loadbalancing/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "Region1-location-name": {
+      "defaultValue": "Central US",
+      "allowedValues": [
```

Comment:
@rkotti The parameter naming style isn't consistent. Some of them follow camelCase and others follow some other type. Can you make this consistent?